README.md

```diff
@@ -28,13 +28,17 @@ as it is middleware, you can also register it for a subset of routes.
 // Start PHP session
 session_start();
 
-$app = new \Slim\App();
+// Create Container
+$container = new Container();
+AppFactory::setContainer($container);
+
+$app = AppFactory::create();
 
 // Register with container
 $container = $app->getContainer();
-$container['csrf'] = function ($c) {
+$container->set('csrf', function (ContainerInterface $container) {
```

Comment:
The `set()` method is actually not part of the interface. Hence why I typically like giving an example with a certain implementation e.g.: PHP-DI.

Reply:
I already use PHP-DI. But as you said, I need to precise which container I use, because of `set()` method.